### PR TITLE
ExUI-659 MV Upgrade to v2.9.5

### PR DESCRIPTION
**Pull request by mwaheedanjum**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,10 +85,10 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.18.3-ConsoleLogs.1",
+    "@hmcts/ccd-case-ui-toolkit": "6.18.2-exui-659-mv-upgrade-rc1",
```
Comment (codaimaster) [inline package.json:88]: version should be incremented either in minor version or patch element.

File: src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.ts [hunk 2]
```diff
@@ -72,4 +74,8 @@ export class MediaViewerWrapperComponent implements OnInit {
   public isIcpEnabled(isEnabled: boolean, icpJurisdictions: string[]): boolean {
     return icpJurisdictions && icpJurisdictions.length > 0 ? icpJurisdictions.includes(this.caseJurisdiction) : isEnabled;
   }
+
+  public isEnableRedactSearch(isEnabled: boolean): boolean {
+    return isEnabled;
+  }
```
Comment (codaimaster) [inline src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.ts:80]: couldn't quite get this one.

File: package.json (hunk 1 above)
Comment (mwaheedanjum) [reply package.json:88]: Can you please explain what that means? Not sure which minor or patch. I got these changes from @RiteshHMCTS PR

Comment (codaimaster) [reply package.json:88]: The ui toolkit version looks to be going from `6.18.3` down to `6.18.2`. In fact, the ui toolkit version should go up to `6.18.4`. The new MV version is containing some new features, and therefore this change will bring some new features to UI Toolkit itself. Therefore, ui toolkit's "minor version" should be incremented.

**Review by codaimaster — CHANGES_REQUESTED** [review]
Please see my comment about the toolkit version, as per semantic versioning conventions.

**Review by andywilkinshmcts — APPROVED** [review]

**Review by OgunyemiO — APPROVED** [review]

**Review by kasi-subramaniam — APPROVED** [review]

**Review by olusegz07 — APPROVED** [review]

**Pull request merged**